### PR TITLE
Chambre gui

### PR DESCRIPTION
**Pull request by ElkySchles**
Screenshot 2023-11-09 at 2 18 05 PM

### CODE REVIEW

**Comment by andrewoid** [comment]
Add a screenshot of the running application.

**Comment by ElkySchles** [comment]
We need a little help figuring out how to reconfigure the dimensions of the elements on the screen.

File: src/main/java/touroll/gui/GuiFrame.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        setContentPane(mainPanel);
+        setSize(700, 500);
+        setTitle("Touroll");
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+        //add the list to the main panel
+        list = new JList<>(listModel);
+        mainPanel.add(list);
+        scrollPane = new JScrollPane(list);
+        mainPanel.add(scrollPane, BorderLayout.WEST);
```
Comment (andrewoid) [inline src/main/java/touroll/gui/GuiFrame.java]: This should be in the CENTER

**Comment by chanacham** [comment]
In planning the UI- we made a zoom whiteboard to plan all the different layouts we are going to be using 
[Elky Schlesinger's Personal Meeting Room 2023-11-09 01_19 PM-[1699555078850].pdf](https://github.com/andrewoid/Touroll/files/13311651/Elky.Schlesinger.s.Personal.Meeting.Room.2023-11-09.01_19.PM-.1699555078850.pdf)

**Review by andrewoid — APPROVED** [review]

**Pull request merged**